### ISSUE TITLE
No emulators for retrogaming

### ISSUE THREAD

**Issue by gsantner** [ehough/docker-kodi#25]
Hi there,

thans for the nice image!

Though packages are missing for retro gaming that were added with kodi 18.
So you can't play any games currently, just profiles are available but no emulator cores.

Can those packages please be added to the image /apt install?

Thanks!

![grafik](https://user-images.githubusercontent.com/6735650/58371423-7fa8b080-7f00-11e9-86c1-df478c77d80a.png)

```
kodi-game-libretro-fbalpha
kodi-game-libretro-fbalpha2012
kodi-game-libretro-fuse
kodi-game-libretro-gambatte
kodi-game-libretro-prboom
kodi-game-libretro-stella
kodi-game-libretro-tgbdual
kodi-game-libretro-vba-next
kodi-game-libretro-virtualjaguar
```

I can also send a PR if preffered

**Comment by ehough** [comment]
Thank you for this report! I haven't seen these packages until now, but I'm all for adding them to the image. I'll do it this week and push out an update; I've been meaning to do that anyway to update to Kodi 18.2.

Stand by ...

**Comment by ehough** [comment]
Fixed in 4128b5e11dc94a71dbc7e92602d2e5b1f35803e8 and now available on Docker Hub, so simply `docker pull erichough/kodi` and you'll have it.

Thanks again!
